.github/CHANGELOG.md

```diff
@@ -137,6 +137,11 @@ This release contains contributions from (in alphabetical order):
   method.
   [(#564)](https://github.com/XanaduAI/strawberryfields/pull/564)
 
+* Cleanup `backends/tfbackend/ops.py` to reduce line count, clarify function
+  similarity across backend ops, and replace `tensorflow.tensordot` with
+  broadcasting.
+  [(#567)](https://github.com/XanaduAI/strawberryfields/pull/567)
```

Comment:
We've recently released 0.18, so this can be moved up to the v0.19 section of the changelog :) Don't forget to add your name to the contribution section there as well!

Reply:
Ready to go! Apologies for the delay 😄